packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -39,6 +39,23 @@ import 'xcode_build_settings.dart';
 import 'xcode_debug.dart';
 import 'xcodeproj.dart';
 
+const String kJITCrashFailureMessage =
+    'Crash occurred when compiling unknown function in unoptimized JIT mode in unknown pass';
+
+const String kJITCrashFailureInstructions = '''
+════════════════════════════════════════════════════════════════════════════════
+A change to iOS has caused a temporary break in Flutter's debug mode on
+physical devices.
+See https://github.com/flutter/flutter/issues/163984 for details.
+
+In the meantime, we recommend these temporary workarounds:
+
+* When developing with a physical device, use one running iOS 18.3 or lower.
+* Use a simulator for development rather than a physical device.
+* If you must use a device updated to iOS 18.4+, use Flutter's release or
```

Comment:
"iOS 18.4+" at the point this string is used do we know the iOS version the device is running, can we string interpolate the current version instead of hard-coding iOS 18.4 since we don't know what iOS version will actually land?
"If you must use a device updated to iOS 19, use Flutter's release or" ...